hedera-node/src/main/java/com/hedera/services/utils/accessors/AccessorFactory.java

```diff
@@ -39,13 +40,38 @@ public AccessorFactory(final AliasManager aliasManager) {
 		this.aliasManager = aliasManager;
 	}
 
-	public PlatformTxnAccessor constructFrom(SwirldTransaction transaction) throws InvalidProtocolBufferException {
-		final var body = extractTransactionBody(Transaction.parseFrom(transaction.getContents()));
-		final var function = functionExtractor.apply(body);
+	public TxnAccessor nonTriggeredTxn(byte[] signedTxnWrapperBytes) throws InvalidProtocolBufferException {
+		final var subtype = constructSpecializedAccessor(signedTxnWrapperBytes);
+		subtype.setTriggered(false);
```

Comment:
Do we actually need a separate `boolean triggered` flag? Or is it enough to check whether `scheduleRef == null`?